runtimes/cloudformation/cmd/handler/main.go

```diff
@@ -77,12 +99,32 @@ func ConfigClosure() interface{} {
 		LogGroup:           logGroup,
 	}
 
+	return configuration
+}
+
+func ConfigClosure() interface{} {
+	configuration := GetConfig()
 	return func(ctx context.Context, event MacroInput) (MacroOutput, error) {
 		return HandleRequest(configuration, ctx, event)
 	}
 }
 
 func main() {
 	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
-	lambda.Start(ConfigClosure())
+
+	switch os.Getenv("KILT_MODE") {
+	case "local":
+		result, err := PatchLocalFile(GetConfig(), context.Background(), os.Getenv("KILT_SRC_TEMPLATE"))
+		if err != nil {
+			panic("cannot patch local file " + os.Getenv("KILT_SRC_TEMPLATE"))
+		}
+
+		err = os.WriteFile(os.Getenv("KILT_OUT_TEMPLATE"), result, 0644)
+		if err != nil {
+			panic("cannot write dst file " + os.Getenv("KILT_OUT_TEMPLATE"))
+		}
+
+	default:
+		lambda.Start(ConfigClosure())
+	}
```

Comment:
Hmm how about we move the GetConfig call before the switch, and then:
* pass the variable into PatchLocalFile
* inline ConfigClosure into the default case (it becomes a single statement)

Reply:
Uh sure